crates/swc_ecma_minifier/tests/projects/output/react-17.0.1.js

```diff
@@ -136,13 +136,14 @@
             case REACT_PROVIDER_TYPE:
                 return getContextName(type._context) + ".Provider";
             case REACT_FORWARD_REF_TYPE:
-                return outerType = type, innerType = type.render, wrapperName = "ForwardRef", functionName = innerType.displayName || innerType.name || "", outerType.displayName || ("" !== functionName ? wrapperName + "(" + functionName + ")" : wrapperName);
```

Comment:
Thish seems to be the only regression -- mainly due to not many people use `switch` today -- which only adds `4 + 1 - 1` byte